authors/apps/articles/urls.py

```diff
@@ -56,4 +57,10 @@
         'bookmarks',
         BookMarkRetrieveAPI.as_view(),
         name='bookmarked'),
+    path(
```

Comment:
@Rhytah, this url should only have method to create and update.
Don't allow list method